pages/linux/cpupower.md

```diff
@@ -0,0 +1,25 @@
+# cpupower
+
+> Tools regarding CPU power and tuning options.
+> This command is available as part of the `cpupower` package, or on Fedora as part of `kernel-tools`.
+> More information: <https://linux.die.net/man/1/cpupower>.
+
+- List CPUs:
```

Comment:
When I run this command, I get
Error parsing cpu list

Arch Linux

Is this the case on other distros?

Reply:
I run Arch too and I've never seen that error. Alternative commands like `sudo cpupower -c all info` (and more verbose: `sudo cpupower -c all frequency-info`) exist and do list the number of each CPU.
![image](https://user-images.githubusercontent.com/64190512/163334790-78880629-9f3f-4f24-a319-1357f9e04b2b.png)

Would it be worth/should I replace `sudo cpupower --cpu cpulist` in the examples with `sudo cpupower --cpu all info`?